src/main/java/com/mongodb/jdbc/Column.java

```diff
@@ -19,15 +16,13 @@ public Column(
             String tableAlias,
             String column,
             String columnAlias,
-            String bsonType,
-            BsonValue value) {
```

Comment:
`Column` now represents just the metadata for a column

Reply:
thanks for this comment!